help/link-to-profile.md

```diff
@@ -0,0 +1,37 @@
+# Link to a profile
+
+Zulip allows you to easily copy a direct link to someone's profile.
+
+## Link to a profile
+
+{start_tabs}
+
+{tab|desktop-web}
+
+{!right-sidebar-view-profile.md!}
+
+1. At the top of the user profile, Click on the **link** (<i class="zulip-icon zulip-icon-link"></i>)
+   located to the right of their name to copy the **user profile link**.
+
```

Comment:
Let's change this to:

> 1. At the top of the user profile, click on the **link** (<i class="zulip-icon zulip-icon-link"></i>) icon
   located to the right of their name to copy a link to their profile.